### ISSUE TITLE
I cant edit astroid template

### ISSUE THREAD

**Issue by mrizgar94** [templaza/astroid-framework#813]
**i cant edit**
when i click this button (template options) show me this page "Internal Server Error
"
**To Reproduce**
1. Go to 'System'
2. Click on 'Site Template Styles'
3. Click on 'click template style'
4. Click on button 'template option'
5. See error  " Internal Server Error "

![problem](https://github.com/user-attachments/assets/0cda22e0-74fd-426f-8afb-5afb1ccfceaa)

**System Informations (please complete the following information):**
 - Joomla Version: [e.g. Joomla 5]
 - PHP Version [e.g. PHP 8]
 - Astroid Version [e.g. Astroid 3.1.0]
 - Browser [e.g. stock browser, google chrom]

**Additional context**
NOTE: but in localhost in xampp i dont have any problem perfect work without problem
but when i upload site in host with database show me this problen in online....

please fix this problem it's important for me thanks

**Comment by sonvnn** [comment]
"Interner server error" related to your server issue. I recommend use Akeeba Backup to transfer site, too. Please try it and give me the result.

**Comment by mrizgar94** [comment]
> Make a backup with Akeeba Backup. The DB is always backed up there. Then use a new empty directory and a new database. Run the installation with Akeeba Kickstart. But that is not a bug.
> 
> https://www.akeeba.com/products/akeeba-backup.html



> Make a backup with Akeeba Backup. The DB is always backed up there. Then use a new empty directory and a new database. Run the installation with Akeeba Kickstart. But that is not a bug.
> 
> https://www.akeeba.com/products/akeeba-backup.html

Okay my bro thanks i will be try

**Comment by mrizgar94** [comment]
> "Internal server error" related to your server issue. I recommend use Akeeba Backup to transfer site, too. Please try it and give me the result.

trasnfer site with "akeeba" ftom localhost to online webhost ?

**Comment by sonvnn** [comment]
@mrizgar94 You can use Akeeba to backup and use its kickstart to restore all of your site to anywhere.

**Comment by mrizgar94** [comment]
Yes, i used akeeba backup..
from localhost to webhost with akeeba
but now all website pages work and panel administrator work without problem,
but astroid editor page doesn't work
When i click "Template Option Button"
[image: step 1.png]

show me this page in second image
[image: step 2.png]
thanks for helping

**Comment by mrizgar94** [comment]
Website domain if you want to see site
wasman.rivingroup.com

**Comment by sonvnn** [comment]
Please try to use 3.1.4-rc2 and retest again https://github.com/templaza/astroid-framework/releases/tag/v3.1.4-rc2

**Comment by mrizgar94** [comment]
okay my bro i will be try this version after that i replay you if fixed or
no
thanks a lot of

**Comment by layalei** [comment]
Any luck??

**Comment by mrizgar94** [comment]
i used this version bro but now my website show me this error
and not fixed my problem

![Image](https://github.com/user-attachments/assets/d573ce35-4252-40c8-900a-d0d1f5326716)

**Comment by sonvnn** [comment]
Please help me enable Joomla debug and send me your screenshot.

**Comment by mrizgar94** [comment]
after enable debug now show me this

![Image](https://github.com/user-attachments/assets/814fc80b-6b0f-4978-a47c-b7972d757fec)

**Comment by felixkat** [comment]
If this is still an issue can you try the latest Joomla and Astroid installs and report back.  Screenshots or a video would be good to help to see what is happening.

**Comment by layalei** [comment]
Hello can you try this solution 
https://github.com/templaza/astroid-framework/issues/845#issuecomment-2483438640

**Comment by mrizgar94** [comment]
yes bro im try with lasted version but not solution this problem <(

**Comment by felixkat** [comment]
> yes bro im try with lasted version but not solution this problem <(

Any luck with getting your system running?

**Comment by felixkat** [comment]
Closing due to no activity / reply.
